release-please-config.json

```diff
@@ -7,7 +7,11 @@
     "packages/extension-sdk": {},
     "packages/hackathon": {},
     "packages/sdk-node": {},
-    "packages/sdk": {},
+    "packages/sdk": {
+      "changelog-sections": [
+        {"type": "chore", "section": "Bug Fixes"}
```

Comment:
this "might" need to have a `"hidden": false` but I think that's the default so we'll see